msg/manual_control_input.msg

```diff
@@ -0,0 +1,52 @@
+uint64 timestamp			# time since system start (microseconds)
```

Comment:
Gosh I hate how we mix tabs and spaces, and how we align stuff.

Reply:
I agree. We once setteled on tabs for indentation spaces for alignment no? I just realized now that it's mixed in this file but only noticed because things were more misaligned than in `manual_control_setpoint`. Now it looks as good as it gets with the mix.